### PR TITLE
Add doc comments for FormOptions defaults

### PR DESCRIPTION
**Pull request by Marusyk**
Add doc comments for FormOptions defaults

Addresses #32216

Please review and let me know if it fits the requirements

### CODE REVIEW

File: src/Http/Http/src/Features/FormOptions.cs [hunk 1]
```diff
@@ -16,83 +16,124 @@ public class FormOptions
...
+        /// <value>
+        /// Defaults to <c>65,536 bytes‬</c>, which is approximately 0.52MB.
+        /// </value>
         public const int DefaultMemoryBufferThreshold = 1024 * 64;
 
         /// <summary>
         /// Default value for <see cref="BufferBodyLengthLimit"/>.
         /// </summary>
+        /// <value>
+        /// Defaults to <c>134,217,728 bytes‬</c>, which is approximately 1.07GB.
```
Comment (Tratcher) [inline src/Http/Http/src/Features/FormOptions.cs:28]: 1024 * 1024 is 1MB
```suggestion
        /// Defaults to <c>134,217,728 bytes‬</c>, which is 128MB.
```

File: src/Http/Http/src/Features/FormOptions.cs [hunk 2]
```diff
@@ -16,83 +16,124 @@ public class FormOptions
         /// <summary>
         /// Default value for <see cref="MemoryBufferThreshold"/>.
         /// </summary>
+        /// <value>
+        /// Defaults to <c>65,536 bytes‬</c>, which is approximately 0.52MB.
```
Comment (Tratcher) [inline src/Http/Http/src/Features/FormOptions.cs:20]: 1024 is 1KB
```suggestion
        /// Defaults to <c>65,536 bytes‬</c>, which is approximately 64KB.
```

File: src/Http/Http/src/Features/FormOptions.cs [hunk 3]
```diff
@@ -16,83 +16,124 @@ public class FormOptions
         /// <summary>
         /// Default value for <see cref="MemoryBufferThreshold"/>.
         /// </summary>
+        /// <value>
```
Comment (Tratcher) [inline src/Http/Http/src/Features/FormOptions.cs:19]: Do we know if `<value>` shows up in intellisense? Or only the summary?

**Review by Tratcher — DISMISSED** [review]

File: src/Http/Http/src/Features/FormOptions.cs (hunk 3 above)
Comment (Tratcher) [reply src/Http/Http/src/Features/FormOptions.cs:19]: Looks like it doesn't show up, please move these values to the summary.

File: src/Http/Http/src/Features/FormOptions.cs [hunk 4]
```diff
@@ -16,83 +16,124 @@ public class FormOptions
...
+        /// Defaults to <c>128 bytes‬</c>.
+        /// </value>
         public const int DefaultMultipartBoundaryLengthLimit = 128;
 
         /// <summary>
         /// Default value for <see cref="MultipartBodyLengthLimit "/>.
         /// </summary>
+        /// <value>
+        /// Defaults to <c>134,217,728 bytes‬</c>, which is approximately 1.07GB.
```
Comment (Tratcher) [inline src/Http/Http/src/Features/FormOptions.cs:44]: Please update the other values.

File: src/Http/Http/src/Features/FormOptions.cs (hunk 3 above)
Comment (Marusyk) [reply src/Http/Http/src/Features/FormOptions.cs:19]: The text is ok (Default , Defaults)? 
```
Default value for MemoryBufferThreshold. 
Defaults to 65,536 bytes‬, which is approximately 64KB.
```

Comment (JamesNK) [reply src/Http/Http/src/Features/FormOptions.cs:19]: For these const values, lets have the format:

`Default value for MemoryBufferThreshold. 65,536 bytes‬, which is approximately 64KB.`

File: src/Http/Http/src/Features/FormOptions.cs [hunk 5]
```diff
@@ -16,83 +16,124 @@ public class FormOptions
...
+        /// Defaults to <c>65,536 bytes‬</c>, which is approximately 64KB.
+        /// </value>
         public const int DefaultMemoryBufferThreshold = 1024 * 64;
 
         /// <summary>
         /// Default value for <see cref="BufferBodyLengthLimit"/>.
         /// </summary>
+        /// <value>
+        /// Defaults to <c>134,217,728 bytes‬</c>, which is 128MB.
+        /// </value>
```
Comment (JamesNK) [inline src/Http/Http/src/Features/FormOptions.cs:29]: Example:

```suggestion
        /// <summary>
        /// Default value for <see cref="BufferBodyLengthLimit"/>. <c>134,217,728 bytes‬</c>, which is 128MB.
        /// </summary>
```

File: src/Http/Http/src/Features/FormOptions.cs [hunk 6]
```diff
@@ -16,83 +16,124 @@ public class FormOptions
...
         public bool BufferBody { get; set; } = false;
 
         /// <summary>
         /// If <see cref="BufferBody"/> is enabled, this many bytes of the body will be buffered in memory.
         /// If this threshold is exceeded then the buffer will be moved to a temp file on disk instead.
         /// This also applies when buffering individual multipart section bodies.
         /// </summary>
+        /// <value>
+        /// Defaults to <c>65,536 bytes‬</c>, which is approximately 0.52MB.
+        /// </value>
```
Comment (JamesNK) [inline src/Http/Http/src/Features/FormOptions.cs:63]: Still have "Defaults to" with the settable properties.

```suggestion
        /// <summary>
        /// If <see cref="BufferBody"/> is enabled, this many bytes of the body will be buffered in memory.
        /// If this threshold is exceeded then the buffer will be moved to a temp file on disk instead.
        /// This also applies when buffering individual multipart section bodies.
        /// Defaults to <c>65,536 bytes‬</c>, which is approximately 0.52MB.
        /// </summary>
```

File: src/Http/Http/src/Features/FormOptions.cs (hunk 3 above)
Comment (JamesNK) [reply src/Http/Http/src/Features/FormOptions.cs:19]: Yes, that is good.

File: src/Http/Http/src/Features/FormOptions.cs [hunk 7]
```diff
@@ -16,83 +16,124 @@ public class FormOptions
         /// <summary>
         /// Default value for <see cref="MemoryBufferThreshold"/>.
         /// </summary>
+        /// <value>
+        /// Defaults to <c>65,536 bytes‬</c>, which is approximately 64KB.
```
Comment (JamesNK) [inline src/Http/Http/src/Features/FormOptions.cs:20]: Could you remove the `<c></c>` tags? I don't think our other default docs have these. Typically they are only used for keywords like `<c>null</c>` or `<c>false</c>`.

**Review by JamesNK — APPROVED** [review]
LGTM!

Thanks for helping out with this.

**Review by Tratcher — APPROVED** [review]

**Pull request merged**